src/core/ext/filters/client_channel/lb_policy/pick_first/pick_first.cc

```diff
@@ -218,6 +261,13 @@ void PickFirst::AttemptToConnectUsingLatestUpdateArgsLocked() {
   if (latest_update_args_.addresses.ok()) {
     addresses = *latest_update_args_.addresses;
   }
+  if (latest_update_args_.config != nullptr) {
```

Comment:
As per out internal conversation, it looks like we actually want to move this back to `UpdateLocked()`.  Looks like this can go in an `else` block on line 277.